### DISCUSSION TITLE
Xiaomi Yi-hacks-v5

### DISCUSSION THREAD

**Discussion by husnainali09** [blakeblackshear/frigate#3632] [Ask A Question]
What would be best settings for these yihacks in config. I know a few people here are running them so wondering whats best practise.

**Comment by NickM-27** [comment]
In general we have had a number of users have issues with them, but https://github.com/blakeblackshear/frigate/issues/2858 should provide some good info on setting it up / config challenges and how they were overcome

**Reply by husnainali09** [reply]
Thanks I will reference from there

**Comment by husnainali09** [comment]
I looked at that thread and it doesn't really have any specific yaml for xiaomis. 

Issue I'm facing is that every 2-3 days, all the xiaomi cameras all at once go white (hik and dahua keep working) and in debug it shows an earlier view so looks like the process on frigate side just halts for these xiaomi yihacks after a few days. Any thoughts boys

**Reply by NickM-27** [reply]
what do the logs say? that would be helpful to know
